src/components/AdminPortal/AddPeopleToTag/AddPeopleToTag.module.css

```
@@ -104,14 +122,38 @@
+  padding: var(--space-2) var(--space-3);
   border-radius: var(--radius-lg);
-  box-shadow: 0 var(--shadow-offset-xs) var(--shadow-blur-sm)
-    var(--color-gray-300);
+  background: var(--color-gray-200);
+  color: var(--color-gray-700);
+  font-size: var(--font-size-md);
+  font-weight: var(--font-weight-semibold);
+  line-height: var(--line-height-tight);
+}
+
+.removeMemberChipButton:global(.btn) {
```

Stylelint (17.6.0)

[error] Unexpected unknown pseudo-class selector ":global" (selector-pseudo-class-no-unknown)

(selector-pseudo-class-no-unknown)